main/src/ca/uwaterloo/flix/language/phase/typer/TypeReduction.scala

```diff
@@ -335,6 +344,15 @@ object TypeReduction {
     }
   }
 
+  private def getField(clazz: Class[_], fieldName: String): Option[Field] = {
```

Comment:
Add comments

Reply:
Can this be put next to the other retrieveField or?

Reply:
I will make a follow up pr to this file - remove unused variables, rearrange methods, etc